app/controllers/auth/helpers/formatDuetime.js

```diff
@@ -0,0 +1,9 @@
+const moment = require('moment')
```

Comment:
You don't need moment, we are already using date-fns, can you please that instead?